webserver.yml

```diff
@@ -15,15 +12,15 @@
     - name: geerlingguy.apache
       when: islandora_build_base_box|bool == True
     - name: geerlingguy.php-versions
-      when: islandora_build_base_box|bool == True
+      when: islandora_build_base_box|bool == False
     - name: geerlingguy.php
-      when: islandora_build_base_box|bool == True
+      when: islandora_build_base_box|bool == False
     - name: geerlingguy.php-mysql
-      when: islandora_build_base_box|bool == True
+      when: islandora_build_base_box|bool == False
     - name: geerlingguy.git
       when: islandora_build_base_box|bool == True
     - name: geerlingguy.composer
-      when: islandora_build_base_box|bool == True
+      when: islandora_build_base_box|bool == False
```

Comment:
Did you intend to move PHP and Composer to the second step, or was this for easier testing? (either's ok, just checking)

Reply:
Ah! it is causing a problem. My playbook breaks while creating the base box. At line 18 of crayfish.yml we try to install composer, but we haven't installed php yet.

Reply:
I ran into the same issue. I switched them back:
```diff
diff --git a/webserver.yml b/webserver.yml
index 171d9e9..28fb86c 100644
--- a/webserver.yml
+++ b/webserver.yml
@@ -12,11 +12,11 @@
     - name: geerlingguy.apache
       when: islandora_build_base_box|bool == True
     - name: geerlingguy.php-versions
-      when: islandora_build_base_box|bool == False
+      when: islandora_build_base_box|bool == True
     - name: geerlingguy.php
-      when: islandora_build_base_box|bool == False
+      when: islandora_build_base_box|bool == True
     - name: geerlingguy.php-mysql
-      when: islandora_build_base_box|bool == False
+      when: islandora_build_base_box|bool == True
     - name: geerlingguy.git
       when: islandora_build_base_box|bool == True
     - name: geerlingguy.composer
```
Now it appears to work as advertised.